pages/index.js

```diff
@@ -103,6 +118,21 @@ export default function HomePage({ onToggleFavourites, favouriteIdeas }) {
               </CardListItem>
             ))}
       </CardList>
+      <ButtonBox>
+        <PreviousButton
+          onClick={() => setCurrentPage(currentPage - 1)}
+          show={currentPage > 1}
+        >
+          Previous Page
+        </PreviousButton>
+
+        <NextButton
+          onClick={() => setCurrentPage(currentPage + 1)}
+          show={currentPage < totalPages}
+        >
+          Next Page
+        </NextButton>
+      </ButtonBox>
```

Comment:
Why two different names since they have the same styles? ^^

Also the display logic can be put just in React like:
```jsx
<ButtonBox>
{ (currentPage > 1) ? <button onClick={...}>Prev</button> : null } 
{ (currentPage < totalPages) ? <button onClick={...}>Next</button> : null } 
</ButtonBox>
```